packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```diff
@@ -195,7 +199,14 @@ export const SendFundsDialog: FC<SendFundsDialogProps> = ({ onExitClick, account
             }}
           />
         </FormField>
-        <FormField label="Destination account" error={errors.account?.message}>
+        <FormField
+          label="Destination account"
+          error={errors.account?.message}
+          tooltip={{
+            text: 'Any Polkadot wallet address format is supported, but if you’re transferring tokens over to another Joystream member, we recommend using the Joystream wallet address format, which starts with “J4”.',
```

Comment:
Let's use computed `joystreamAddressPrefix` here instead of hardcoded value. Also, I think we should use regular qutoes `""`, instead of `“”`, that's what we do in the rest of the app